cr-examples/onnx/src/main/java/oracle/code/onnx/OnnxInterpreter.java

```diff
@@ -57,7 +58,7 @@ public static Object interpret(Class<? extends OnnxOp> opClass,
                               .map(i -> (Tensor)(i instanceof Optional o ? o.get() : i))
                               .toList(),
                     schema.outputs().size(),
-                    attributeMap);
+                    attributeMap, Arena.ofAuto());
```

Comment:
Using an explicit arena here would cause just too much ripple effects in all the auto-generated op classes